spec/errors.md

```diff
@@ -278,28 +278,28 @@ for example when encountering a value with an unsupported type
 or an internally inconsistent set of options.
 
 > For example, attempting to format any of the following messages
-> might result in a _Formatting Error_ if done within a context that
->
-> 1. provides for the variable reference `$user` to resolve to
->    an object `{ name: 'Kat', id: 1234 }`,
-> 2. provides for the variable reference `$field` to resolve to
->    a string `'address'`, and
-> 3. uses a `:get` formatting function which requires its argument to be an object and
->    an option `field` to be provided with a string value,
+> might result in a _Formatting Error_ because the `operand` type doesn't
+> match the expectations of the formatting function:
```

Comment:
The changes made to this section don't really seem beneficial, as they get somewhat stuck in the nitty-gritty of how exactly our built-in datetime functions work. The earlier examples were rather purposefully abstract.

Reply:
I disagree? The point of this section is to show different examples of formatting errors.

- The first shows a type mismatch in a single expression.
- The second shows a type mismatch with a local value.
- The third is the same as the second, except that it uses a non-string type for the local value.
- The last shows an internal error (attempting to format a time value as a date).

Admittedly these are related to what our built-in date/time functions do, but they are, after all, _built-in_ and thus could be familiar to anyone. I might include more text or maybe a couple of examples. Note that these avoid _resolution_ errors, such as assigning an invalid value to an option or using an unknown option or unknown function.